shared/types/api/node.go

```diff
@@ -52,6 +52,9 @@ type NodeStatusResponse struct {
 	EthMatchedLimit                          *big.Int        `json:"ethMatchedLimit"`
 	PendingMatchAmount                       *big.Int        `json:"pendingMatchAmount"`
 	CreditBalance                            *big.Int        `json:"creditBalance"`
+	CreditAndBalance                         *big.Int        `json:"creditAndBalance"`
```

Comment:
Probably want to call this like `CreditAndDonatedBalance` or something since the above is `CreditBalance`, it sounds weird if it's just `CreditAndBalance`... Not gonna die on that hill though. Same to the other fields below.

Reply:
I was keeping it consistent with names used on the smart contracts. But will change here.